src-docs/src/views/guidelines/writing_examples.js

```diff
@@ -0,0 +1,730 @@
+            <EuiModalHeader>
+              <EuiModalHeaderTitle>Remove index pattern?</EuiModalHeaderTitle>
+            </EuiModalHeader>
+
+            <EuiModalFooter>
+              <EuiButtonEmpty>Cancel</EuiButtonEmpty>
+
+              <EuiButton fill color="danger">
+                Remove
+              </EuiButton>
+            </EuiModalFooter>
+          </EuiPanel>
+        </GuideRuleExample>
+      </GuideRule>
+
+      <GuideRule
+        heading=""
+        description="Create separate confirmations for single and bulk actions."
+      >
+        <GuideRuleExample
+          type="do"
+          text="Asking users to delete a single item should include the item name in the title, if possible.
+   Use single quotes around the name if it helps clarify meaning."
+        >
+          <EuiPanel paddingSize="none" style={modalStyles}>
+            <EuiModalHeader>
+              <EuiModalHeaderTitle>
+                Delete pipeline &apos;MyPipeline&apos;?
+              </EuiModalHeaderTitle>
+            </EuiModalHeader>
+            <EuiModalBody>
+              <EuiText>
+                <p>You can&apos;t recover deleted data.</p>
+              </EuiText>
+            </EuiModalBody>
+
+            <EuiModalFooter>
+              <EuiButtonEmpty>Cancel</EuiButtonEmpty>
+
+              <EuiButton color="danger" fill>
+                Delete
+              </EuiButton>
+            </EuiModalFooter>
+          </EuiPanel>
+        </GuideRuleExample>
+
+        <GuideRuleExample
+          type="do"
+          text="For bulk actions, include the number of items in the title."
+        >
+          <EuiPanel paddingSize="none" style={modalStyles}>
+            <EuiModalHeader>
+              <EuiModalHeaderTitle>Delete 6 pipelines?</EuiModalHeaderTitle>
+            </EuiModalHeader>
+            <EuiModalBody>
+              <EuiText>
+                <p>You can&apos;t recover deleted data.</p>
+              </EuiText>
+            </EuiModalBody>
+
+            <EuiModalFooter>
+              <EuiButtonEmpty>Cancel</EuiButtonEmpty>
+
+              <EuiButton color="danger" fill>
+                Delete
+              </EuiButton>
+            </EuiModalFooter>
+          </EuiPanel>
+        </GuideRuleExample>
+      </GuideRule>
+
+      <EuiHorizontalRule />
+
+      <EuiSpacer size="m" />
+
+      <EuiText>
+        <h2 id="switches">Switches</h2>
+
+        <p>Describe the action when the switch is enabled.</p>
+      </EuiText>
+
+      <GuideRule heading="" description="">
+        <GuideRuleExample
+          panelDisplay="block"
+          type="do"
+          text="Keep labels and descriptions short and direct."
+        >
+          <EuiSwitch label="Use the first matching geo data" />
+          <EuiSpacer />
+        </GuideRuleExample>
+        <GuideRuleExample
+          panelDisplay="block"
+          type="do"
+          text='Start with a keyword instead of "If true" or "If enabled".'
```

Comment:
In this case, I'm not sure which "keyword" is meant in the "use recommended defaults" switch. Or was the rollover text supposed to be part of the switch?
```suggestion
          text='Start with a keyword instead of "If true" or "If enabled".'
```